README.md

```diff
@@ -124,6 +124,24 @@ Configure attributes:
 
 This will enable the SFTP Server and chroot every user in the `sftpusers` group to the `/home/sftp/%u` directory.
 
+## Extra Configuration Options
```

Comment:
@bdwyertech cool, thanks for adding this to the docs. Can you please add something like 

```
....
* `['ssh-hardening']['ssh']['server']['extras']` - `{}`. Add extra configuration options, see [below](#extra-configuration-options) for details
....
* `['ssh-hardening']['ssh']['client']['extras']` - `{}`. Add extra configuration options, see [below](#extra-configuration-options) for details
```

to the attribute documentation above?

Can you do me a favor and cleanup a commit history by [squashing](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) the commits?

Then we can merge it :-)